contracts/pcv/maker/IMakerRouter.sol

```diff
@@ -0,0 +1,68 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity =0.8.13;
+
+interface IMakerRouter {
+    /// @notice Function to swap from FEI to DAI
+    /// @param amountFeiIn the amount of FEI to be deposited
+    /// @param minDaiAmountOut the minimum amount of DAI expected to be received
+    /// @param to the address the DAI should be sent to once swapped
+    function swapFeiForDai(
+        uint256 amountFeiIn,
+        uint256 minDaiAmountOut,
+        address to
+    ) external;
+
+    /// @notice Function to swap all of FEI balance to DAI
+    /// @param to the address the DAI should be sent to once swapped
+    function swapAllFeiForDai(address to) external;
+
+    /// @notice Function to swap from FEI to USDC
+    /// @dev Function will swap from FEI to DAI first then DAI to USDC
+    /// @param amountFeiIn the amount of FEI to be deposited
+    /// @param minDaiAmountOut the minimum amount of DAI expected to be received
+    /// @param to the address the DAI should be sent to once swapped
+    function swapFeiForUsdc(
+        uint256 amountFeiIn,
+        uint256 minDaiAmountOut,
+        address to
+    ) external;
+
+    /// @notice Function to swap all of FEI balance to USDC
+    /// @param to the address the USDC should be sent to once swapped
+    function swapAllFeiForUsdc(address to) external;
+
+    /// @notice Function to swap for both DAI and USDC
+    /// @dev Function will swap from FEI to DAI first then DAI to USDC
+    /// @param amountFeiIn the amount of FEI to be deposited
+    /// @param minDaiAmountOut the minimum amount of DAI expected to be received
+    /// @param ratioUSDC the ratio of the DAI received we would like to swap to USDC - in basis point terms
+    /// @param usdcTo the address the USDC should be sent to once swapped
+    /// @param daiTo the address the DAI should be sent to once swapped
+    function swapFeiForUsdcAndDai(
+        uint256 amountFeiIn,
+        uint256 minDaiAmountOut,
+        address usdcTo,
+        address daiTo,
+        uint256 ratioUSDC
+    ) external;
+
+    /// @notice Function to swap all FEI balance for both DAI and USDC
+    /// @param usdcTo the address the USDC should be sent to once swapped
+    /// @param daiTo the address the DAI  should be sent to once swapped
+    /// @param ratioUSDC the ratio of the DAI received we would like to swap to USDC - in basis point terms
+    function swapAllFeiForUsdcAndDai(
+        address usdcTo,
```

Comment:
We don't mind about slippage in this case?

Reply:
Slippage is automatically prevented with the minDaiAmountOut parameter which is calculated based on the balance https://github.com/volt-protocol/volt-protocol-core/pull/97/files#diff-cdf022fcb8c0c46a72ee2f87f0c08154095eacedc7f3d1106829f0b9e59cc0dcR189 an internal function takes the amount being swapped, multiplies it by 99.97%, and requires that at least that amount comes out, otherwise it will revert and prevent slippage.